packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -663,12 +663,10 @@ class RenderListWheelViewport
   /// by [childManager].
   @override
   void performLayout() {
-    final BoxConstraints childConstraints =
-      constraints.copyWith(
-        minHeight: _itemExtent,
-        maxHeight: _itemExtent,
-        minWidth: 0.0,
-      );
```

Comment:
Why move this? It does not seem to change later in the course of laying out.

Reply:
Just move to where first use it.